src/Microsoft.OData.Edm/ExtensionMethods/ExtensionMethods.cs

```diff
@@ -252,6 +253,41 @@ public static IEnumerable<IEdmVocabularyAnnotation> FindVocabularyAnnotationsInc
             return result;
         }
 
+        /// <summary>
+        /// Gets a target path's vocabulary annotations defined in a specific model and models referenced by that model.

```

Comment:
Curious about "models referenced by that model" part...
Do we have any tests to verify that this method is able to retrieve vocabulary annotations defined in referenced models?

Reply:
This methods calls a method that retrieves annotations from the current model and referenced models. I assume that is already properly tested